### PR TITLE
#305 Moving constants to a common repo.

### PR DESCRIPTION
**Pull request by JAYATEJA04**
### What is the change?

- Made an npm package to store all the important constant variables in the join.js file. Importing variables is done by installing the npm package. To install the package, run `npm i www-constants` in the terminal. After installing the package, the variables are imported by using the "require" keyword. The package which store the variables is named "www-constants".
- All the constants which are commented on are the ones which have been stored in the npm package.

The link to the repository of the package is <a href="https://github.com/JAYATEJA04/www-constants">www-constants</a>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,5 +15,8 @@
   },
   "pre-commit": [
     "check"
-  ]
+  ],
+  "dependencies": {
+    "www-constants": "^1.0.2"
```
Comment (rohan09-raj) [inline package.json:20]: Where this dependency is located?

File: js/join.js [hunk 2]
```diff
@@ -20,82 +22,92 @@ const flowState = {
...
+//   state: 1,
+//   country: 1,
+//   skills: 5,
+//   college: 1,
+//   introduction: 100,
+//   whyRds: 100,
+//   forFun: 100,
+//   funFact: 100,
+//   foundFrom: 1,
+// };
```
Comment (rohan09-raj) [inline js/join.js:43]: Why the code is being commented here?

File: js/join.js [hunk 3]
```diff
@@ -20,82 +22,92 @@ const flowState = {
...
 
 let url;
+// const postUrl = 'https://api.realdevsquad.com/users/self/intro';
 
-const inputFields = document.querySelectorAll('input');
-const textAreas = document.querySelectorAll('textarea');
+// const inputFields = document.querySelectorAll('input');
+// const textAreas = document.querySelectorAll('textarea');
+//
```
Comment (rohan09-raj) [inline js/join.js:50]: Why the code is being commented here?

File: js/join.js [hunk 4]
```diff
@@ -20,82 +22,92 @@ const flowState = {
...
-const city = document.getElementById('city');
-const state = document.getElementById('state');
-const country = document.getElementById('country');
-const next1 = document.getElementById('next1');
+// const city = document.getElementById('city');
+// const state = document.getElementById('state');
+// const country = document.getElementById('country');
+// const next1 = document.getElementById('next1');
+// const previous1 = document.getElementById('previous1');
+//
```
Comment (rohan09-raj) [inline js/join.js:70]: Why the code is being commented here?

File: js/join.js [hunk 5]
```diff
@@ -20,82 +22,92 @@ const flowState = {
...
-const funFact = document.getElementById('funFact');
-const next2 = document.getElementById('next2');
+// const introduction = document.getElementById('introduction');
+// const skills = document.getElementById('skills');
+// const college = document.getElementById('college');
+// const forFun = document.getElementById('forFun');
+// const funFact = document.getElementById('funFact');
+// const next2 = document.getElementById('next2');
+// const previous2 = document.getElementById('previous2');
+//
```
Comment (rohan09-raj) [inline js/join.js:86]: Why the code is being commented here?

File: js/join.js [hunk 6]
```diff
@@ -20,82 +22,92 @@ const flowState = {
...
+// const previewIntro = document.getElementById('previewIntro');
+// const previewSkills = document.getElementById('previewSkills');
+// const previewInstitution = document.getElementById('previewInstitution');
+// const previewFunFact = document.getElementById('previewFunFact');
+// const previewForFun = document.getElementById('previewForFun');
+// const previewWhyRds = document.getElementById('previewWhyRds');
+// const previewHeardAbout = document.getElementById('previewHeardAbout');
+// const previous4 = document.getElementById('previous4');
+// const submit = document.getElementById('next4');
+//
```
Comment (rohan09-raj) [inline js/join.js:110]: Why the code is being commented here?

**Review by RitikJaiswal75 — CHANGES_REQUESTED** [review]
Why is my join.js file being changed it will break the join section. As per the requirements we were looking for an npm package that can have all the constants and we can use them everywhere.

**Comment by RitikJaiswal75** [comment]
Please do not use a feature that is already created, create a new POC and hide it behind a feature flag

**Comment by rohan09-raj** [comment]
Closing this PR as this has been continued in a separate new repository 
https://github.com/JAYATEJA04/realdevsquad-constants
Message Reference: https://discord.com/channels/673083527624916993/1007355818603192360/1069837282095419413

**Pull request closed without merging**